### ISSUE TITLE
Corrupted font embedding into PDF documents

### ISSUE THREAD

**Issue by RaySub** [alif-type/libertinus#302]
Hello,

I have a problem to embed the Bold and Bold Italic variants of Libertinus Sans into a PDF document.
The characters don't show up though the fonts are said to be incoroprated in the document's properties.

I could reproduce the issue using two different PDF virtual printers. The first one is CutePDF+Ghostscript 9.26 and the second one is edocPDFprinter by Iteksoft (proprietary engine, not based on Ghostscript). No such problem with the italic and regular flavours, nor with any member of the Libertinus Serif family.

I am working on Windows 10 and my source document is edited using Word (Office 365).

**Comment by khaledhosny** [comment]
Can you reproduce this with other Windows apps e.g. WordPad? (I don’t have Word). Please also attach the broken PDF.

**Comment by RaySub** [comment]
Hi Khaled,

This had nothing to do with the Libertinus font family.

I ended up by removing the complete family using the font management tool of Windows. Then, I launched Powershell, listed the C:\Windows\Fonts and C:\Users\myusername\AppData\Local\Microsoft\Windows\Fonts directories and removed any libertinus*.otf$$ duplicate files. I restarted the computer and everything goes fine now.

Kind Regards,
Raymond
